gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -1034,11 +1034,10 @@
      * @param toStepLabel the step label of the incoming vertex
      * @return the traversal with the modified {@link AddEdgeStep}
      * @since 3.1.0-incubating
-     * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#addedge-step" target="_blank">Reference Documentation - AddEdge Step</a>
      */
     public default GraphTraversal<S, E> to(final String toStepLabel) {
         this.asAdmin().getBytecode().addStep(Symbols.to, toStepLabel);
-        ((AddEdgeStep) this.asAdmin().getEndStep()).addTo(__.select(toStepLabel));
+        ((FromToModulating) this.asAdmin().getEndStep()).addTo(toStepLabel);
```

Comment:
but you still use `from()` and `to()` with `addE()` right? we just now use it with path related stuff as well, so it relates to all of those things correct? I guess I'm saying that the javadoc should reference to the all the steps that `from()` and `to()` modulate.